cmd/main.go

```diff
@@ -185,9 +186,10 @@ func middleware(log *slog.Logger, cfg *config.Config, po *persisted_operations.P
 	aliases.NewMaxAliasesRule(cfg.MaxAliases)
 	tks := tokens.MaxTokens(cfg.MaxTokens)
 	vr := ValidationRules(schema, tks, cfg.ObfuscateValidationErrors)
+	disableMethod := disable_method.DisableMethodRule(cfg.DisableGetMethod)
```

Comment:
maybe just call it "disableGET" since thats the only thing it disables? also that's what its called in the config?

Reply:
Applied suggestion